apps/PiKISS GUI/uninstall

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+#Allow packages required by this app to be uninstalled
+purge_packages || exit 1
```

Comment:
Your uninstall script does not sufficiently uninstall the app. Unless I'm mistaken, multiple folders will be left over, unless your `./build.sh` deleted them all already.

Reply:
Some commits were not pushed online.

Fixed.